static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -104,6 +105,15 @@ function StacktraceLinkSetup({organization, project, event}: StacktraceLinkSetup
   );
 }
 
+export function isMobileLanguage(event: Event) {
+  return (
+    isMobilePlatform(event.platform) ||
+    (event.platform === 'other' &&
```

Comment:
This is what we currently have under the [other platform](https://sentry.io/organizations/sentry/discover/results/?field=stacktrace_link.file&field=sdk.name&field=count%28%29&name=Other+platform+%28fixes+needed%29&project=1&query=transaction%3A%2Fapi%2F0%2Fprojects%2F%7Borganization_slug%7D%2F%7Bproject_slug%7D%2Fstacktrace-link%2F++stacktrace_link.platform%3Aother&sort=-stacktrace_link.file&statsPeriod=7d&yAxis=count%28%29).

If you drill down on `main.dart` you will see that is mostly the sentry-sdks org.